### DISCUSSION TITLE
Does working_tree_dir actually raise an error?

### DISCUSSION THREAD

**Discussion by Greedquest** [gitpython-developers/GitPython#1510] [Q&A]
https://github.com/gitpython-developers/GitPython/blob/183cf3509f2a1646daaca1874594a27abb408deb/git/repo/base.py#L332-L334

It looks like this function does not raise any error. But in the documentation is says it should raise AssertionError

https://github.com/gitpython-developers/GitPython/blob/183cf3509f2a1646daaca1874594a27abb408deb/git/repo/base.py#L108-L112

Am I going crazy or does this not work as advertised?

**Comment by Greedquest** [comment]
Looks like it was removed 8 years ago in e4d3809

https://github.com/gitpython-developers/GitPython/commit/e4d3809161fc54d6913c0c2c7f6a7b51eebe223f#diff-3cc1aaf2f1e2bc1341d3f71ceec44b2762b981280b4d162e26327bd558721fe1

Probably either fix the docs or restore the error

**Reply by Byron** [reply]
Thanks for letting me know! Since the current behaviour has been present for a couple of years already, we wouldn't want to change it and instead adapt the documentation. Is this something you would like to contribute?

Thanks a lot.
